app/controllers/spree/api/v2/storefront/account_confirmations_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        class AccountConfirmationsController < ::Spree::Api::V2::BaseController
+
+          def show
+            user = Spree.user_class.confirm_by_token(params[:id])
+
+            if user.errors.empty?
+              head :ok
```

Comment:
We should know if the user needs to wait for manual account approval or can log in immediately. For example, with a response like `{"status": "active"}` and `{"status": "manual"}`. @pmilewski @damianlegawiec

Reply:
I added a response with the state.